### ISSUE TITLE
Only add these fields on the meetings post type

### ISSUE THREAD

**Issue by corsacca** [thecodezone/disciple-tools-three-thirds#7]
https://github.com/thecodezone/disciple-tools-three-thirds/blob/787342049867dfb2ccd89880c6dada500b274a7d/meeting-type/meeting-type.php#L101

**Comment by corsacca** [comment]
Seeing these fields on contacts, groups, etc

**Comment by micahmills** [comment]
@corsacca can you confirm that this is still a problem. I was taking a look at it for @incraigulous and I can't duplicate what you were seeing. There have been some updates to the Meetings plugin that may have fixed this issue though.

**Comment by corsacca** [comment]
@micahmills 
Yeah, since there is no post type protection, these fields are added to each post type:
![image](https://user-images.githubusercontent.com/24901539/199685911-63a1c577-9c32-4398-9c9d-b0b163403b50.png)

**Comment by incraigulous** [comment]
This issue will be fixed once https://github.com/thecodezone/disciple-tools-three-thirds/pull/8 is released.
